File: app/src/main/java/com/androideas/quakesnz/app/ui/AboutActivity.java
package com.androideas.quakesnz.app.ui;

import android.os.Bundle;
import android.support.v7.app.ActionBarActivity;
import android.text.util.Linkify;
import android.widget.TextView;

import com.androideas.quakesnz.app.R;

public class AboutActivity extends ActionBarActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        Linkify.addLinks((TextView) findViewById(R.id.about_body), Linkify.ALL);
        Linkify.addLinks((TextView) findViewById(R.id.acknowledgements_body), Linkify.ALL);
    }
}

File: app/src/main/java/nz/co/codebros/quakesnz/about/AboutActivity.kt
package nz.co.codebros.quakesnz.about

import android.os.Bundle
import android.text.util.Linkify
import androidx.appcompat.app.AppCompatActivity
import nz.co.codebros.quakesnz.databinding.ActivityAboutBinding

class AboutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAboutBinding.inflate(layoutInflater)
        setContentView(binding.root)
        Linkify.addLinks(binding.aboutBody, Linkify.ALL)
        Linkify.addLinks(binding.acknowledgementsBody, Linkify.ALL)
    }
}
